flink-autoscaler/src/main/java/org/apache/flink/autoscaler/event/AutoScalerEventHandler.java

```diff
@@ -83,12 +92,27 @@ default void handleScalingEvent(
     }
 
     static String scalingReport(
-            Map<JobVertexID, ScalingSummary> scalingSummaries, boolean scalingEnabled) {
-        StringBuilder sb =
-                new StringBuilder(
-                        scalingEnabled
-                                ? SCALING_SUMMARY_HEADER_SCALING_ENABLED
-                                : SCALING_SUMMARY_HEADER_SCALING_DISABLED);
+            Map<JobVertexID, ScalingSummary> scalingSummaries,
+            boolean scalingEnabled,
+            boolean isExcluded,
+            Configuration config) {
+        StringBuilder sb = new StringBuilder();
+        if (!scalingEnabled) {
+            sb.append(
+                    String.format(
+                            SCALING_SUMMARY_HEADER_SCALING_EXECUTION_DISABLED,
+                            SCALING_ENABLED.key(),
+                            false));
+        } else if (isExcluded) {
+            sb.append(
+                    String.format(
+                            SCALING_SUMMARY_HEADER_SCALING_EXECUTION_DISABLED,
+                            EXCLUDED_PERIODS.key(),
+                            config.get(EXCLUDED_PERIODS)));
+        } else {
+            sb.append(SCALING_SUMMARY_HEADER_SCALING_EXECUTION_ENABLED);
+        }
```

Comment:
as I wrote in the other comment, I prefer not to make any changes in the event handler

Reply:
I think it would make the code much cleaner to simply submit a string to the event handler instead of the 2 booleans.

Reply:
Modify the parameter of method `handleScalingEvent`  to replace the scalingEnabled and scalingBlocked flags via a generic String